README.md

```diff
@@ -21,21 +21,19 @@ Use `fzf.fish` to interactively find and insert different shell entities into th
 - **Search input:** recursive listing of current directory's non-hidden files
 - **Key binding and mnemonic:** <kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>F</kbd> (`F` for file)
 - **Preview window:** file with syntax highlighting, directory contents, or file type
-- **Remarks**
+- **Special features**
   - appends `/` if the selection is a directory and the only path selected so you can hit <kbd>ENTER</kbd> to [immediately cd into it][cd docs]
   - if the current token is a directory with a trailing slash (e.g. `.config/<CURSOR>`), then that directory is searched instead
   - ignores files that are also ignored by git
-  - <kbd>Tab</kbd> to multi-select
 
 ### Modified paths
 
 ![gif git status](images/git_status.gif)
 
 - **Search input:** the current repository's `git status`
 - **Key binding and mnemonic:** <kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>S</kbd> (`S` for status)
-- **Remarks:** <kbd>Tab</kbd> to multi-select
 
-### A commit hash
+### Commit hashes
```

Comment:
This and Shell variables header should have been plural to represent the ability to select multiple of them